### PR TITLE
Add FindCommandIntegrationTest #334

### PR DESCRIPTION
**Pull request by limmlingg**
Part of #327

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * Contains integration tests (interaction with the Model) for {@code FindCommand}.
+ */
+public class FindCommandIntegrationTest {
+    private Model model;
+    private TypicalPersons persons = new TypicalPersons();
+    private FindCommandParser parser = new FindCommandParser();
+
+    @Before
+    public void setUp() {
+        model = new ModelManager(persons.getTypicalAddressBook(), new UserPrefs());
```
Comment (Zhiyuan-Amos) [inline src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java]: remove this method. See https://github.com/se-edu/addressbook-level4/pull/495#pullrequestreview-44817270

File: src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        FindCommand command = prepareCommand("Bob", model);
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
+
+        assertCommandSuccess(command, expectedMessage, Collections.emptyList());
+    }
+
+    /**
+     * Generates a new {@code FindCommand} by parsing the {@code userInput} and updating {@code model}
+     */
+    private FindCommand prepareCommand(String userInput, Model model) throws Exception {
```
Comment (Zhiyuan-Amos) [inline src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java]: i don't think there's a need for `prepareCommand` to take in a `Model`. :>

File: src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    public void execute_multiplePersonsFound_success() throws Exception {
+        FindCommand command = prepareCommand("Meier", model);
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 2);
+
+        assertCommandSuccess(command, expectedMessage, Arrays.asList(persons.benson, persons.daniel));
+    }
+
+    @Test
+    public void execute_noPersonFound_success() throws Exception {
+        FindCommand command = prepareCommand("Bob", model);
```
Comment (Zhiyuan-Amos) [inline src/test/java/seedu/address/logic/commands/FindCommandIntegrationTest.java]: `Bob` sounds too legit of a name. Maybe something like `NoSuchPerson` would do better. Or `Foo`?

**Comment by yamgent** [comment]
Also no CanIHasReview. :(

**Comment by limmlingg** [comment]
Continued in #513

**Pull request closed without merging**